src/da65/scanner.c

```diff
@@ -413,6 +413,9 @@ void InfoNextTok (void)
         case '\"':
             NextChar ();
             I = 0;
+            if (C == '\"') {
+                InfoSVal[0] = '\0';
```

Comment:
You don't need to use that test.  `InfoSVal` always should be preset to an empty string there.

Reply:
Fixed